public/v4/apps/node-exporter.yml

```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: prom/node-exporter:$$cap_version
+        restart: always
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock:ro
+            - /sys:/host/sys:ro
+            - /proc:/host/proc:ro
+            - /:/rootfs:ro
+        logging:
```

Comment:
Logging is not consumed by CapRover, I'll remove after merge.
![image](https://user-images.githubusercontent.com/10582082/210267192-072e47c6-eccf-485e-ae96-f1aabe4ff066.png)